app/webpack/javascripts/modules/external_users/claims/Dropzone.js

```diff
@@ -161,19 +161,23 @@ moj.Modules.Dropzone = {
         const fileId = response.document.id
 
         this.createDocumentIdInput(response.document.id)
-        tableRow.replaceWith(this.notificationHTML(fileName, 'success', 'File has been uploaded.', fileId))
+        tableRow.replaceWith(this.notificationHTML(fileName, 'govuk-tag govuk-tag--green', 'Uploaded', fileId))
         this.status.html(response.document.filename + ' has been uploaded.')
       }.bind(this),
 
       error: function (xhr, status, error) {
         const fileName = file.name
+        const errorMessage = xhr.responseJSON ? xhr.responseJSON.error : xhr.responseText
 
         if (status === 'timeout') {
-          tableRow.replaceWith(this.notificationHTML(fileName, 'error', 'The server failed to process your file.'))
-          this.status.html('The server failed to process your file.')
+          tableRow.replaceWith(this.notificationHTML(fileName, 'govuk-tag govuk-tag--red word-wrap', 'Upload timed out'))
```

Comment:
One minor refactor you could make to remove some repetition could be to define `govuk-tag govuk-tag--red word-wrap` as a `const` at the start of the `uploadFile` function (eg `const govukError = 'govuk-tag govuk-tag--red word-wrap'`) 

You can then replace the three uses of it in this block of code, so 
`tableRow.replaceWith(this.notificationHTML(fileName, 'govuk-tag govuk-tag--red word-wrap', 'Upload timed out'))` 

becomes 

`tableRow.replaceWith(this.notificationHTML(fileName, govukError, 'Upload timed out'))` etc.

This means if we want to change that style in future we only need to do it one place.

Reply:
Thanks Mark